File: CodeWars-Solutions/Sum-of-Numbers.py
# Python
def get_sum(a,b):
    #good luck!
    sum = 0
    if (a == b):
        return a
    if (a < b):
        sum = a
        for x in range (a, b):
            sum += (x + 1)
        return sum
    if (b < a):
        sum = b
        for x in range (b, a):
            sum += (x + 1)
        return sum

File: CodeWars-Solutions/Sum-of-Numbers.java
//Java
public class Sum{
  public static void main(String[] args){
    int a = 0;
    int b = 1;
    int sum = 0;
    if (a == b){
      return a;
    }
    if (a < b){
      sum = a;
      for (int i = a; i < b; i++){
        sum += i + 1;
      }
      return sum;
    }
    if (a > b){
      sum = b;
      for (int i = b; i < a; i++){
        sum += i + 1;
      }
      return sum;
    } 
    return sum;
  }
}
